introduction.md

```diff
@@ -1,2 +1,6 @@
 "My name is Sichao"
-"my favorite Codeschool path is JavaScript"
\ No newline at end of file
+"my favorite Codeschool path is JavaScript"
+*Javascript
+*CSS
+*AngularJS
+*Bootstrap
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?